spec/predicates/release.md

```diff
@@ -156,8 +156,55 @@ attestation SHOULD have a subject per artifact:
 ## Changelog and Migrations
 
 As this is the initial version, no changes or migrations to previous versions.
-This proposal is a subset of the information in the existing npm
-[publish attestation], so npm could easily migrate to this specification.
+The required predicates in this specification are a subset of the information
+in the existing npm [publish attestation], so npm could easily migrate to this
+specification.
+
+For example, here is an existing npm publish attestation:
+
+```json
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
```

Comment:
This should probably be Statement/v1. Same below.

Reply:
Oops! I got carried away setting things to v0.1 - I have now fixed this.